content/docs/community/contribution-guide.md

```diff
@@ -0,0 +1,415 @@
+
+7. **Examples for clarity**:
+    - **Do**: Provide clear examples.
+      - *Example*: "For instance, to upload a file, click on the 'Upload' button."
+
+8. **Use US English**:
+    - **Do**: Adhere to US English spelling and grammar rules.
+
+9. **Avoid emojis and exclamations**:
+    - **Don't**: Use emojis or exclamation marks in the documentation.
+
+### Link instead of repeating text
+
+Avoid repeating or duplicating information from other topics. Instead, link to the original source of information and explain why it is important.
+
+### Capitalization
+
+Use lowercase wherever possible.
+
+For topic titles, use sentence-style capitalization. For example: "Create your first project"
+
+Product names should align with the official names of the products, protocols, etc., maintaining exact capitalization.
+
+#### UI text
+
+When referencing specific user interface text, such as button labels or menu items, use the same capitalization displayed in the user interface wherever possible.
+
+#### Feature names
+
+Generally, feature names should be lowercase.
+
+#### Other terms
+
+Capitalize names of:
+
+- Neon tiers and plans. For example, Neon Free Tier, Neon Pro Plan.
+- Third-party organizations, software, and products. Kubernetes, Git, and Vercel.
+- Methods or methodologies. Continuous Integration, Continuous Deployment, etc.
+
+Follow the capitalization style used by the authoritative source, which may use non-standard case styles. For example: PostgreSQL, GitHub, npm.
+
+### Fake user information
+
+When including user information in connection details, API calls, or UI instructions, don't use real user information or email addresses.
+
+- Use an email address ending in `@example.com` or `domain.com`.
+- Use strings like `example_username` or one of the following diverse and non-gendered names: Zhang Kai, Alex Lopez, or Dana Smith.
+
+### Connection strings
+
+Connection strings should be defined as follows:
+
+```text
+postgres://[user]:[password]@[neon_hostname]/[dbname]
+```
+
+If you need to provide a connection string with realistic values, use one of the user names mentioned above, `AbC123dEf` for the password, and `dbname` for the database name:
+
+```text
+postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname
+```
+
+### Commands, parameters, values, filenames
+
+Commands, parameters, values, filenames, error messages, connection strings, and other similar items should be enclosed in backticks. For example:
+
+- "Run the `neonctl projects list` command."
+
+- "Execute 'git clone' to clone a Git repository..."
+
+- 'git clone' is a command that should be in lowercase, whereas Git is the product and should have a capital G.
+
+- "A connection string has this format: `postgres://[user]:[password]@[neon_hostname]/[dbname]`"
+
+## Questions?
+
+If you have questions or run into any issues, please reach out to us on the [Neon Community forum](https://community.neon.tech/).
```

Comment:
If they are docs process questions, will putting those to the community get to the docs team?

Reply:
I think this is OK. If someone asks a question about doc processes and we don't see it right away, I think someone on the team will let us know.